packages/gatsby-theme-docs/src/components/release-notes-subscribe-links.js

```diff
@@ -1,6 +1,56 @@
 import React from 'react';
+import { css } from '@emotion/core';
+import SpacingsStack from '@commercetools-uikit/spacings-stack';
+import SpacingsInline from '@commercetools-uikit/spacings-inline';
+import { designSystem } from '@commercetools-docs/ui-kit';
+import { MailIcon } from '@commercetools-uikit/icons';
+import RssSvg from '../icons/rss.svg';
+import Link from './link';
 
-// TODO: Actual implementation
-const ReleaseNotesSubscribeLinks = () => <div></div>;
+const containerStyle = css`
+  padding-left: ${designSystem.dimensions.spacings.m};
+`;
+
+const linkStyles = css`
+  color: ${designSystem.colors.light.textSecondary} !important;
+  font-size: ${designSystem.typography.fontSizes.body};
+  line-height: ${designSystem.typography.lineHeights.body};
+
+  svg {
+    width: ${designSystem.dimensions.widths.rssIconWidth};
+    * {
+      fill: ${designSystem.colors.light.surfaceSecondary3} !important;
+    }
+  }
+
+  :hover {
+    color: ${designSystem.colors.light.linkNavigation} !important;
+
+    svg {
+      * {
+        fill: ${designSystem.colors.light.linkNavigation} !important;
+      }
+    }
+  }
+`;
+
+const ReleaseNotesSubscribeLinks = () => (
+  <div css={containerStyle}>
```

Comment:
Just use a `styled` component

```suggestion
  <Container>
```